src/streamlink/plugins/youtube.py

```diff
@@ -349,7 +358,12 @@ def _get_streams(self):
         if not is_live:
             streams, protected = self._create_adaptive_streams(info, streams, protected)
 
-        hls_playlist = info.get("hlsvp")
```

Comment:
if youtube doesn't use this anymore,
remove also every other code related to `hlsvp` if it is not used anymore